addons/hr_holidays/models/res_partner.py

```diff
@@ -1,12 +1,20 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from odoo import api, models
+from odoo import api, models, fields
 from odoo.addons.mail.tools.discuss import Store
 
 
 class ResPartner(models.Model):
     _inherit = 'res.partner'
 
+    leave_date_to = fields.Date(compute="_compute_leave_date_to")
+    def _compute_leave_date_to(self):
```

Comment:
missing empty line

Reply:
I did that purposefully; having the compute defined right after the field makes our life easier.
But I guess guideline comes first, so I added the line